dotcom-rendering/cdk/lib/userData.ts

```diff
@@ -22,8 +22,8 @@ export const getUserData = ({
 		`usermod -a -G frontend aws-kinesis-agent-user`,
 		`cd /home/dotcom-rendering`,
 
-		`aws --region eu-west-1 s3 cp s3://aws-frontend-artifacts/frontend/${stage}/${app}/${app}.zip ./`,
-		`unzip -q ${app}.zip -d ${app}`,
+		`aws --region eu-west-1 s3 cp s3://aws-frontend-artifacts/frontend/${stage}/${app}/${app}.tar.gz ./`,
```

Comment:
Can we instead reference the bucket name via the SSM parameter [`/account/services/artifact.bucket` ](https://eu-west-1.console.aws.amazon.com/systems-manager/parameters/account/services/artifact.bucket/description?region=eu-west-1&tab=Table#list_parameter_filters=Name:Contains:artif)? That would align with the [recommendation for S3 bucket names](https://github.com/guardian/recommendations/blob/e6c14c1a4c2c0641e74128bec0d8caf2c7320ac1/github.md#private-information). 

Perhaps a change for upcoming PR and not this one